### ISSUE TITLE
App crashes if folder with the same name exists

### ISSUE THREAD

**Issue by locovna** [4ndv/mrmanga#1]
There is folder with the same name, I downloaded a volume, and want next one be in the same folder.
I suggest, _at least_, write about that option at the instruction or create `folder_that_already_exist_2`

```shell
>>> mrmanga
Welcome to mrmanga 0.4.0!
It will create folders into CURRENT PATH!!
Tell me the link of the manga you want to download (example: http://readmanga.me/your_name)?   http://readmanga.me/kakegurui___compulsive_gambler__ 
Just press Enter to download all the volumes, or specity volumes using this: 1-5 or this: 1, 2, 3, 4, 5 (or 1,2,3,4,5) syntaxes. Also you can download one volume by entering it's number (like this: 1)
Which volumes should i download?  |all|  6
Will download these volumes: 6
How many threads should i use to download? (6)  
Create pdfs for volumes?  |yes|

Add pages in PDF outline?  |no|

Parsing manga info
Parsed, downloading
<Path to gems>/gems/mrmanga-0.4.0/lib/mrmanga/downloader.rb:15:in `initialize': Folder 'kakegurui___compulsive_gambler__' already exists (RuntimeError)
	from <Path to gems>/gems/ruby-2.4.1/gems/mrmanga-0.4.0/lib/mrmanga/cli.rb:59:in `new'
	from <Path to gems>/gems/ruby-2.4.1/gems/mrmanga-0.4.0/lib/mrmanga/cli.rb:59:in `interactive_shell'
	from <Path to gems>/gems/ruby-2.4.1/gems/mrmanga-0.4.0/exe/mrmanga:5:in `<top (required)>'
	from <Path to gems>/gems/ruby-2.4.1/bin/mrmanga:22:in `load'
	from <Path to gems>/gems/ruby-2.4.1/bin/mrmanga:22:in `<main>'
	from <Path to gems>/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
	from <Path to gems>/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by 4ndv** [comment]
Можно на русском :)

Вечером с работы приду - что-нибудь придумаю

**Comment by 4ndv** [comment]
> Можно на русском :)

If you're speaking Russian, ofc

**Comment by 4ndv** [comment]
Fixed in `0.5.0`, you can update your current version with `gem install mrmanga`, thx for report! :3

**Comment by locovna** [comment]
Works perfect! Thanks
